Digital Twin/procedures/createSecureImagePathUrl.vail

```diff
@@ -0,0 +1,8 @@
+PROCEDURE createSecureImagePathUrl(path String)
```

Comment:
Should probably go in a service. Is this only relevant to a specific plugin, like Photo Registration?